workers-site/index.js

```diff
@@ -41,6 +42,13 @@ async function handleEvent(event) {
         bypassCache: true,
       }
     }
+
+    if (url.pathname === "/" || url.pathname === "/index.html"){
+      const page = await getAssetFromKV(event, options)
+
+      return new HTMLRewriter().on('head', new StringInjector("OPERATORS", OPERATORS)).transform(page)
```

Comment:
Why not do the `csvParse` in the worker instead? You'd inject the data structure directly instead of a string, and save on having to include `d3-dsv` as a client dependency.

Reply:
Ended up reworking it so the CSV is parsed at Worker-Deploy-Time

Reply:
In that case we can drop the script tag dep for `d3-dsv`.

Reply:
Removed

Reply:
Cool. We can delete the `d3-dsv.min.js` file from the vendor dir too. No reason to keep it.